pyproject.toml

```
@@ -0,0 +1,46 @@
+[tool.ruff]
+target-version = "py39"
+
+[tool.ruff.lint]
+select = [
+    # Pyflakes
+    "F",
+    # pycodestyle
+    "E",
+    "W",
+    # isort
+    "I",
+    # pep8-naming
+    "N",
+    # flake8-bugbear without opinionated rules
+    "B0",
+    # flake8-pie
+    "PIE",
+    # flake8-print
+    "T20",
+]
+extend-per-file-ignores = { "*/migrations/*" = ["E501"], "*/tests/*" = ["E501"] }
+
+[tool.ruff.lint.pycodestyle]
+max-line-length = 120
+
+[tool.ruff.lint.mccabe]
+max-complexity = 10
+
+[tool.pytest.ini_options]
+DJANGO_SETTINGS_MODULE = "atv.settings"
+norecursedirs = [".git", "venv*", "var"]
+doctest_optionflags = ["NORMALIZE_WHITESPACE", "IGNORE_EXCEPTION_DETAIL", "ALLOW_UNICODE"]
+
+[tool.coverage.run]
+branch = true
+omit = ["*migrations*", "*site-packages*", "*venv*", "*tests*"]
+
+
+[tool.ruff.lint.isort]
+combine-as-imports = true
+order-by-type = false
+extra-standard-library = ["token", "tokenize", "enum", "importlib"]
+known-first-party = ["atv", "documents", "services", "utils"]
+known-third-party = ["django", "six"]
```

```suggestion
[tool.pytest.ini_options]
DJANGO_SETTINGS_MODULE = "atv.settings"
norecursedirs = [".git", "venv*", "var"]
doctest_optionflags = ["NORMALIZE_WHITESPACE", "IGNORE_EXCEPTION_DETAIL", "ALLOW_UNICODE"]

[tool.coverage.run]
branch = true
omit = ["*migrations*", "*site-packages*", "*venv*", "*tests*"]
```
We'll want our projects to be as similar to each other as possible so there shouldn't be any project-specific Ruff configurations unless absolutely needed.